flow/scripts/io_placement.tcl

```diff
@@ -1,12 +1,16 @@
 source $::env(SCRIPTS_DIR)/load.tcl
 load_design 3_1_place_gp_skip_io.odb 2_floorplan.sdc "Starting io placement"
 
-if {[info exists ::env(IO_CONSTRAINTS)]} {
-  source $::env(IO_CONSTRAINTS)
-}
-place_pins -hor_layer $::env(IO_PLACER_H) \
+if {[info exists ::env(FLOORPLAN_DEF)]} {
+    puts "Use DEF in to initialization floorplan, Skip io placement."
```

Comment:
This sounds like a command.  Perhaps

Skipping io placement as a DEF was used during floorplan initialization to set the io placement.